test/typed_buffers_test.dart

```diff
@@ -17,21 +17,26 @@ main() {
   });
   testUint(16, (l) => new Uint16Buffer(l));
   testInt(16, (l) => new Int16Buffer(l));
-  testUint(32, (l) => new Uint32Buffer(l));  /// 01: ok
+  testUint(32, (l) => new Uint32Buffer(l));
+
+  /// 01: ok
```

Comment:
This looks very wrong.
The /// should be changed to //# instead, and kept on the original line.

Alternatively, they can just be removed. Since this was moved out of the SDK repo, I don't think we use the SDK test runner to run it anyway.

More of these below.

Reply:
ack